src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/localvariablename/InputLocalVariableNameOneCharInitVarName.java

```diff
@@ -12,7 +12,7 @@
 {
 	public void fooMethod()
 	{
-		for(int i = 1; i <10; i++) {
+		for(int i = 1, J = 1; i <10; i++) {
```

Comment:
update in tests is very small, I expected more updates.

Please provide links to lines in Input files which covers all cases that we discussed in issue and you have in doc examples.

Reply:
I added test cases for capital letters in for init statement in for-each and usual for.
Documentation was updated with similar examples